packages/components/src/Card/Card.tsx

```diff
@@ -46,13 +45,12 @@ interface HeaderActionProps {
   readonly title?: string;
   readonly action?: ButtonAction;
 }
-interface ButtonAction {
-  readonly label: string;
-  readonly size?: "small" | "base" | "large";
-  readonly type?: "primary" | "secondary" | "tertiary";
-  readonly icon?: IconNames;
-  readonly onClick?: () => void;
-}
+
+type ButtonAction = Pick<
+  ButtonProps,
+  "label" | "size" | "type" | "icon" | "onClick"
```

Comment:
Is there any reason not to allow all Button props here? `variation` in particular, but also options like `external`,  `disabled`, the full suite of `aria{Property}`, etc.

Reply:
I'm also leaning towards that idea, however am uncertain if this would be better (friendlier) to make `Card` take smaller set of parameters